documentation.md

```diff
@@ -37,6 +36,14 @@ Use `\horizontalLine`.
 
 Create a link with `\externalLink{text}{URL}`.
 
+## Licence
+
+Add licence with `\licence[path/to/image]{Licence name}{URL}` 
+
+## Inline source code 
+
+Create a inline source code with `\CodeInline[bash]{make test}`
```

Comment:
Oh, I didn't see last time, but that should, then, be `\codeInline` (because of the coding style). Also, add a comment so that we know that special caracters shoudl **not** be escaped in this case (so that something like `\CodeInline[bash]{cd ${TMP%xy}z && make test_inline}` is perfectly valid, dont't ask me by which magic)